components/chainhook-sdk/src/observer/http.rs

```diff
@@ -222,34 +196,32 @@ pub fn handle_new_stacks_block(
             prometheus_monitoring.stx_metrics_block_appeneded(new_tip);
             let background_job_tx = background_job_tx.inner();
             match background_job_tx.lock() {
-                Ok(tx) => {
-                    let _ = tx.send(ObserverCommand::PropagateStacksChainEvent(chain_event));
-                }
+                Ok(tx) => match tx.send(ObserverCommand::PropagateStacksChainEvent(chain_event)) {
+                    Ok(_) => {}
+                    Err(e) => {
+                        return error_response(
+                            format!("Unable to send stacks chain event: {}", e.to_string()),
+                            ctx,
+                        );
+                    }
+                },
```

Comment:
Please check the comment above about ignoring success cases and nested statements.